[APM] Mark old api test client as legacy

sorenlouv opened the pull request:
This PR contains two changes:
 - Move towards the builtin roles `editor` and `viewer` defined in https://github.com/elastic/kibana/issues/90806
 - Prefix deprecated api services (`legacySupertestAsApmReadUser`) to Urge APM devs to use the type-safe `apmApiClient` service instead


## Old client vs type safe client

**Old client**
```ts
const supertest = getService("supertestAsApmReadUser");

// returns an untyped response
const response = supertest.get({
  pathname: `/api/apm/services/opbeans-node/error_groups/main_statistics`,
  query: {
    start: range.start,
    end: range.end,
    transactionType: 'request',
    environment: "ENVIRONMENT_ALL",
    kuery: "",
  },
});
```

**New client**
```ts
const apmApiClient = getService("apmApiClient");

// returns a typed response, and will type check the arguments as well
const { response, status } = apmApiClient.readUser({
  endpoint: `GET /api/apm/services/{serviceName}/error_groups/main_statistics`,
  params: {
    path: { serviceName: "opbeans-node" },
    query: {
      start: range.start,
      end: range.end,
      transactionType: 'request',
      environment: "ENVIRONMENT_ALL",
      kuery: "",
    },
  },
});
```

sorenlouv committed:
[APM] Mark old api client as legacy

sorenlouv committed:
Fix test

sorenlouv committed:
Fix tsc

smith reviewed (approved):
Looks nice!

justinkambic reviewed (approved):
LGTM

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150783/)
* Commit: 8871f478b72056cc2792a118000cff581eb25246
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Build Kibana](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150783/execution/node/377/log)

### History
* :broken_heart: [Build #150542](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150542/) failed 7dd116c3a512ce5d8524db3f5daf819169d1bfc3

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150542/","number":"150542","commit":"7dd116c3a512ce5d8524db3f5daf819169d1bfc3"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150783/","number":"150783","commit":"8871f478b72056cc2792a118000cff581eb25246"}]}
PIPELINE-->

sorenlouv committed:
Explicit type annotation

Fixes this problem:

info Building TypeScript projects refs for tsconfig.refs.json...
 info [tsc] > node_modules/typescript/bin/tsc -b tsconfig.refs.json --pretty
 proc [tsc] x-pack/test/apm_api_integration/configs/index.ts:37:14 - error TS7056: The inferred type of this node exceeds the maximum length the compiler will serialize. An explicit type annotation is needed.
 proc [tsc]
 proc [tsc] 37 export const configs = mapValues(apmFtrConfigs, (value, key) => {
 proc [tsc]                 ~~~~~~~
 proc [tsc]
 proc [tsc]
 proc [tsc] Found 1 error.
 proc [tsc]
ERROR Unable to build TS project refs

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151034/)
* Commit: 14766678a32eaa01bd4f73302f5afa7dc8ed8ab7
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Build Kibana](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151034/execution/node/370/log)

### History
* :broken_heart: [Build #150783](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150783/) failed 8871f478b72056cc2792a118000cff581eb25246
* :broken_heart: [Build #150542](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150542/) failed 7dd116c3a512ce5d8524db3f5daf819169d1bfc3

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150542/","number":"150542","commit":"7dd116c3a512ce5d8524db3f5daf819169d1bfc3"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150783/","number":"150783","commit":"8871f478b72056cc2792a118000cff581eb25246"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151034/","number":"151034","commit":"14766678a32eaa01bd4f73302f5afa7dc8ed8ab7"}]}
PIPELINE-->

sqren commented:
@elasticmachine merge upstream

kibanamachine committed:
Merge branch 'master' into api-tests-legacy

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151094/)
* Commit: 5a044cbbfb2a2ed38924e0b278cd9860efcf6fef
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Build Kibana](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151094/execution/node/371/log)

### History
* :broken_heart: [Build #151034](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151034/) failed 14766678a32eaa01bd4f73302f5afa7dc8ed8ab7
* :broken_heart: [Build #150783](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150783/) failed 8871f478b72056cc2792a118000cff581eb25246
* :broken_heart: [Build #150542](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150542/) failed 7dd116c3a512ce5d8524db3f5daf819169d1bfc3

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150542/","number":"150542","commit":"7dd116c3a512ce5d8524db3f5daf819169d1bfc3"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150783/","number":"150783","commit":"8871f478b72056cc2792a118000cff581eb25246"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151034/","number":"151034","commit":"14766678a32eaa01bd4f73302f5afa7dc8ed8ab7"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151094/","number":"151094","commit":"5a044cbbfb2a2ed38924e0b278cd9860efcf6fef"}]}
PIPELINE-->

sqren commented:
@elasticmachine merge upstream

kibanamachine committed:
Merge branch 'master' into api-tests-legacy

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151129/)
* Commit: 301df1cf09e43aed6fc048d16583531f44cb4b52
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Build Kibana](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151129/execution/node/371/log)

### History
* :broken_heart: [Build #151094](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151094/) failed 5a044cbbfb2a2ed38924e0b278cd9860efcf6fef
* :broken_heart: [Build #151034](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151034/) failed 14766678a32eaa01bd4f73302f5afa7dc8ed8ab7
* :broken_heart: [Build #150783](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150783/) failed 8871f478b72056cc2792a118000cff581eb25246
* :broken_heart: [Build #150542](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150542/) failed 7dd116c3a512ce5d8524db3f5daf819169d1bfc3

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150542/","number":"150542","commit":"7dd116c3a512ce5d8524db3f5daf819169d1bfc3"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150783/","number":"150783","commit":"8871f478b72056cc2792a118000cff581eb25246"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151034/","number":"151034","commit":"14766678a32eaa01bd4f73302f5afa7dc8ed8ab7"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151094/","number":"151094","commit":"5a044cbbfb2a2ed38924e0b278cd9860efcf6fef"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151129/","number":"151129","commit":"301df1cf09e43aed6fc048d16583531f44cb4b52"}]}
PIPELINE-->

dgieselaar commented on x-pack/test/apm_api_integration/common/apm_api_supertest.ts:
```diff
@@ -60,3 +57,8 @@ Body: ${JSON.stringify(res.body)}`
+
+export interface SupertestReturnType<TEndpoint extends APIEndpoint> {
+  status: number;
+  body: NonNullable<APIReturnType<TEndpoint>>;
```
Why is `NonNullable` necessary here?

sqren replied:
That's what we had in the tests already (moved that over to this util). But will remove it and see what breaks.

dgieselaar commented on x-pack/test/apm_api_integration/common/config.ts:
```diff
@@ -47,24 +68,45 @@ export function createTestConfig(config: Config) {
-        supertestAsApmWriteUser: supertestAsApmUser(servers.kibana, ApmUser.apmWriteUser),
-        supertestAsApmAnnotationsWriteUser: supertestAsApmUser(
+
+        supertestClients: async (context: InheritedFtrProviderContext) => {
```
when is this function called? on every test run, or for every individual test?

dgieselaar replied:
and do you think we can maybe drop the `supertest` part and name it something more appropriately? e.g. `apmApiClient`?

sqren replied:
Renamed to `apmApiClients`

sqren replied:
> when is this function called? on every test run, or for every individual test?

Will be called when the service is loaded:

```
const apmApiClient = getService('apmApiClient');
```

sorenlouv committed:
Fix async problems

sorenlouv committed:
Rename supertestClients to apmApiClients

sorenlouv committed:
Remove redundant NonNullable

sorenlouv committed:
Rename to apmApiClient

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151270/)
* Commit: 7ca0f25d119dc41f52cab4620dc941e8de0390f6
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-110974/7ca0f25d119dc41f52cab4620dc941e8de0390f6)
* This comment will update when the build is complete

### Failed CI Steps
* [Jest Unit Tests](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151270/execution/node/489/log)
* [Execute xpack-kibana-ciGroup6](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151270/execution/node/721/log)

### History
* :broken_heart: [Build #151129](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151129/) failed 301df1cf09e43aed6fc048d16583531f44cb4b52
* :broken_heart: [Build #151094](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151094/) failed 5a044cbbfb2a2ed38924e0b278cd9860efcf6fef
* :broken_heart: [Build #151034](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151034/) failed 14766678a32eaa01bd4f73302f5afa7dc8ed8ab7
* :broken_heart: [Build #150783](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150783/) failed 8871f478b72056cc2792a118000cff581eb25246
* :broken_heart: [Build #150542](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150542/) failed 7dd116c3a512ce5d8524db3f5daf819169d1bfc3

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150542/","number":"150542","commit":"7dd116c3a512ce5d8524db3f5daf819169d1bfc3"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150783/","number":"150783","commit":"8871f478b72056cc2792a118000cff581eb25246"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151034/","number":"151034","commit":"14766678a32eaa01bd4f73302f5afa7dc8ed8ab7"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151094/","number":"151094","commit":"5a044cbbfb2a2ed38924e0b278cd9860efcf6fef"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151129/","number":"151129","commit":"301df1cf09e43aed6fc048d16583531f44cb4b52"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151270/","number":"151270","commit":"7ca0f25d119dc41f52cab4620dc941e8de0390f6"}]}
PIPELINE-->

sorenlouv committed:
Move security service init up

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151306/)
* Commit: e340b9d6f27596fdea778446d3de25647cde31dc
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-110974/e340b9d6f27596fdea778446d3de25647cde31dc)
* This comment will update when the build is complete

### Failed CI Steps
* [Execute xpack-kibana-ciGroup6](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151306/execution/node/718/log)
* [Execute xpack-kibana-ciGroup6](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151306/execution/node/850/log)

### History
* :broken_heart: [Build #151270](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151270/) failed 7ca0f25d119dc41f52cab4620dc941e8de0390f6
* :broken_heart: [Build #151129](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151129/) failed 301df1cf09e43aed6fc048d16583531f44cb4b52
* :broken_heart: [Build #151094](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151094/) failed 5a044cbbfb2a2ed38924e0b278cd9860efcf6fef
* :broken_heart: [Build #151034](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151034/) failed 14766678a32eaa01bd4f73302f5afa7dc8ed8ab7
* :broken_heart: [Build #150783](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150783/) failed 8871f478b72056cc2792a118000cff581eb25246

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/150783/","number":"150783","commit":"8871f478b72056cc2792a118000cff581eb25246"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151034/","number":"151034","commit":"14766678a32eaa01bd4f73302f5afa7dc8ed8ab7"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151094/","number":"151094","commit":"5a044cbbfb2a2ed38924e0b278cd9860efcf6fef"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151129/","number":"151129","commit":"301df1cf09e43aed6fc048d16583531f44cb4b52"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151270/","number":"151270","commit":"7ca0f25d119dc41f52cab4620dc941e8de0390f6"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151306/","number":"151306","commit":"e340b9d6f27596fdea778446d3de25647cde31dc"}]}
PIPELINE-->

sqren commented:
retest

sqren commented:
@elasticmachine merge upstream

kibanamachine committed:
Merge branch 'master' into api-tests-legacy

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151361/)
* Commit: fe3dace036d8a3b1b83efc7bca59a4a0b6166e06
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-110974/fe3dace036d8a3b1b83efc7bca59a4a0b6166e06)
* [Documentation Changes](https://kibana_110974.docs-preview.app.elstc.co/diff)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

### History
* :broken_heart: [Build #151306](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151306/) failed e340b9d6f27596fdea778446d3de25647cde31dc
* :broken_heart: [Build #151270](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151270/) failed 7ca0f25d119dc41f52cab4620dc941e8de0390f6
* :broken_heart: [Build #151129](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151129/) failed 301df1cf09e43aed6fc048d16583531f44cb4b52
* :broken_heart: [Build #151094](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151094/) failed 5a044cbbfb2a2ed38924e0b278cd9860efcf6fef
* :broken_heart: [Build #151034](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151034/) failed 14766678a32eaa01bd4f73302f5afa7dc8ed8ab7

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151034/","number":"151034","commit":"14766678a32eaa01bd4f73302f5afa7dc8ed8ab7"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151094/","number":"151094","commit":"5a044cbbfb2a2ed38924e0b278cd9860efcf6fef"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151129/","number":"151129","commit":"301df1cf09e43aed6fc048d16583531f44cb4b52"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151270/","number":"151270","commit":"7ca0f25d119dc41f52cab4620dc941e8de0390f6"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151306/","number":"151306","commit":"e340b9d6f27596fdea778446d3de25647cde31dc"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151361/","number":"151361","commit":"fe3dace036d8a3b1b83efc7bca59a4a0b6166e06"}]}
PIPELINE-->

kibanamachine commented:
## 💚 Backport successful

| Status | Branch | Result |
|:------:|:------:|:------:|
| ✅ |  [7.x](https://github.com/elastic/kibana/pull/111307)  | [<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/111307">](https://github.com/elastic/kibana/pull/111307) |

This backport PR will be merged automatically after passing CI.

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151377/)
* Commit: fe3dace036d8a3b1b83efc7bca59a4a0b6166e06
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-110974/fe3dace036d8a3b1b83efc7bca59a4a0b6166e06)
* [Documentation Changes](https://kibana_110974.docs-preview.app.elstc.co/diff)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

### History
* :green_heart: [Build #151361](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151361/) succeeded fe3dace036d8a3b1b83efc7bca59a4a0b6166e06
* :broken_heart: [Build #151306](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151306/) failed e340b9d6f27596fdea778446d3de25647cde31dc
* :broken_heart: [Build #151270](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151270/) failed 7ca0f25d119dc41f52cab4620dc941e8de0390f6
* :broken_heart: [Build #151129](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151129/) failed 301df1cf09e43aed6fc048d16583531f44cb4b52
* :broken_heart: [Build #151094](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151094/) failed 5a044cbbfb2a2ed38924e0b278cd9860efcf6fef

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151094/","number":"151094","commit":"5a044cbbfb2a2ed38924e0b278cd9860efcf6fef"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151129/","number":"151129","commit":"301df1cf09e43aed6fc048d16583531f44cb4b52"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151270/","number":"151270","commit":"7ca0f25d119dc41f52cab4620dc941e8de0390f6"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151306/","number":"151306","commit":"e340b9d6f27596fdea778446d3de25647cde31dc"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151361/","number":"151361","commit":"fe3dace036d8a3b1b83efc7bca59a4a0b6166e06"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/151377/","number":"151377","commit":"fe3dace036d8a3b1b83efc7bca59a4a0b6166e06"}]}
PIPELINE-->
